### PR TITLE
CASMHMS-5522: NCN Remove changed stop REDS

### PR DESCRIPTION
**Pull request by shunr-hpe**
## Summary and Scope

Changed the documentation to have the user shutdown cray-reds before
doing the remove. And added a note that the BSS restart may timeout if
there are only 2 ncn workers.

There was a race condition where cray-reds would add back a component
to HSM while the remove_management_ncn.py was removing the component
from BSS, SLS, and HSM.

Added a default ipmi username to ncn_status.py. This makes the script
match the documentation.

## Issues and Related PRs

* Resolves [CASMHMS-5522](https://jira-pro.its.hpecorp.net:8443/browse/CASMHMS-5522)

## Testing

Tested by running the script for all the NCNs on the given hardware. Tested with IPMI_PASSWORD set and without it set.

### Tested on:

The ncn_status.py changes were tested on

  * baldar - csm 1.0.1
  * fanta - csm 1.2.0-beta.104
  * wasp - csm 1.2.0-beta.102

### Test description:

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? No, NA 
- Were continuous integration tests run? If not, why? No, NA
- Was upgrade tested? If not, why? No, NA
- Was downgrade tested? If not, why? No, NA
- Were new tests (or test issues/Jiras) created for this change? No, there are not tests for these scripts

## Risks and Mitigations

The risks are low. The documentation changes are small. The ncn_status.py script does not modify the system. It only provides information while doing the NCN remove.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by rsjostrand-hpe — APPROVED** [review]

File: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md [hunk 1]
```diff
@@ -78,7 +81,13 @@ Remove NCN data to System Layout Service (SLS), Boot Script Service (BSS) and Ha
 
     **Important**: Save the `ifnames` and `bmc_mac` information if you plan to add this NCN back at some time in the future.
 
-1.  Remove the node from SLS, HSM, and BSS:
+1. Shutdown cray-reds:
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md:84]:
```suggestion
1. Shutdown `cray-reds`:
```

File: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md [hunk 2]
```diff
@@ -104,7 +113,30 @@ Remove NCN data to System Layout Service (SLS), Boot Script Service (BSS) and Ha
     Successfully removed x3000c0s26b0n0 - ncn-s004
     ```
 
-1.  Verify the results by fetching the status of the management nodes:
+    **NOTE:**
+    If workers have been removed and the worker count is currently at two, a time out restarting cray-bss can be ignored. For example, the following failure output from `remove_management_ncn.py` can be ignored.
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md:117]:
```suggestion
    If workers have been removed and the worker count is currently at two, a timeout restarting `cray-bss` can be ignored. For example, the following failure output from `remove_management_ncn.py` can be ignored.
```

File: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md [hunk 3]
```diff
@@ -104,7 +113,30 @@ Remove NCN data to System Layout Service (SLS), Boot Script Service (BSS) and Ha
...
+
+    ```screen
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md:119]:
```suggestion
    ```text
```

File: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md [hunk 4]
```diff
@@ -104,7 +113,30 @@ Remove NCN data to System Layout Service (SLS), Boot Script Service (BSS) and Ha
...
+             stdout:
+    Waiting for deployment "cray-bss" rollout to finish: 0 out of 3 new replicas have been updated...
+    Waiting for deployment "cray-bss" rollout to finish: 1 out of 3 new replicas have been updated...
+    Waiting for deployment "cray-bss" rollout to finish: 2 out of 3 new replicas have been updated...
+
+             stderr:
+    error: timed out waiting for the condition
+    ```
+
+1. Start cray-reds:
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md:133]:
```suggestion
1. Start `cray-reds`:
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
Does this also need PRs for 1.2 and main?

**Comment by shunr-hpe** [comment]
> Does this also need PRs for 1.2 and main?

For 1.2 these changes are in the larger Add/Remove/Replace NCN PR:
https://github.com/Cray-HPE/docs-csm/pull/1536

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**